syft/workers/base.py

```diff
@@ -119,6 +121,7 @@ def __init__(
         self.auto_add = auto_add
         self._message_pending_time = message_pending_time
         self.msg_history = []
+        self.trash = {}
```

Comment:
Q: Wouldn't this be a better attribute for the ```storage``` class?

Reply:
Great idea!